setup.cfg

```
@@ -31,6 +31,7 @@ install_requires =
     pyyaml
     sdcflows >= 2.0.0rc5, < 3.0
     smriprep >= 0.8.0rc0 
+    svgutils != 0.3.2
```

```suggestion
    svgutils < 0.3.2
```
As per the conversation in niworkflows, although 0.3.3 will not have the issue it will break other parts of the interface that we are using. I guess there's no other option than pinning this version (or implement the new interface and pin > 0.3.2, when that is released).